### DISCUSSION TITLE
Improve compatibilty (ios)

### DISCUSSION THREAD

**Discussion by ryanbr** [uBlockOrigin/uAssets#18635] [General]
Just a request from a Brave dev,

`Just that requestIdleCallback is unsupported in safari so if we use these scripts in iOS this causes errors. we need to default to setTimeout when requestIdleCallback is undefined`

https://github.com/brave/adblock-resources/pull/115

**Comment by gorhill** [comment]
I don't see a request in there. It's concerning a "deAmp" scriptlet, which we do not have here.

**Reply by ryanbr** [reply]
Will confirm

**Comment by ryanbr** [comment]
https://github.com/brave/uBlock/blob/076c831bb17fc2acd94dde7d5ac4954b673efaa8/assets/resources/scriptlets.js#L1377C25-L1377C25
